### ISSUE TITLE
Very wide confidence interval for predicted trajectories with lcmm

### ISSUE THREAD

**Issue by dianahendrickx** [CecileProust-Lima/lcmm#288]
After fitting trajectories, I got very a very wide confidence interval for one of the classes. It's just a bit smaller than the interval [0,1].
Any idea what can cause this? I use gridsearch with rep=100 and maxiter = 30, and lcmm with maxiter = 200 (other settings default).
If I set maxiter = 100 for lcmm instead of 200, the model does not converge.

**Comment by VivianePhilipps** [comment]
Hi,

we had a similar question in issue #265. Does our previous answer help you ?

 Viviane

**Comment by dianahendrickx** [comment]
Yes, this helps. Furthermore, I like to add that in my case, when I increased the number of initial values from rep=100 to a higher number, I could get a better solution with smaller confidence intervals. For a complex model like a 3rd order model (3rd order terms for age) and 4 classes, setting rep=1500 resulted in a better global optimum with smaller confidence intervals.

**Comment by VivianePhilipps** [comment]
OK. You may have reached a local solution on the first run.
Please note however, that to choose the best solution (for a fixed number of latent classes and a specific trajectory for age), you should look at the log-likelihoods and keep the solution that has the highest log-likelihood.
